sdk/node/src/connection.js

```diff
@@ -5,6 +5,53 @@ const { fetch } = require('fetch-ponyfill')()
 const errors = require('./errors')
 const btoa = require('btoa')
 
+const blacklistAttributes = [
+  'asset_tags',
+  'asset_definition',
+  'account_tags',
+  'reference_data',
```

Comment:
Should probably add:

- `tags`: used in assets.create, accounts.create
- `next`, `after`: pagination cursor in response and request, respectively

We could probably refer to the Swagger spec for any more instances of arbitrary objects.